sparrowpy/radiosity_fast/geometry.py

```diff
@@ -337,6 +398,589 @@ def calculate_normals(points: np.ndarray):
 
     return normals
 
+###################################################
+# integration
+################# 1D , polynomial
+def _poly_estimation_Lagrange(x: np.ndarray, y: np.ndarray) -> np.ndarray:
```

Comment:
a lot of the following things seems to be related to form_factor, maybe shift to formfactor/geometry.py?